components/screens/post/PostScreen.tsx

```diff
@@ -68,6 +80,11 @@ function PostScreen({
     return <LoadingView />;
   }
 
+  const onEndReached = React.useCallback(
```

Comment:
We don't need this. Right now, Lemmy doesn't recognize the page number in the request for comments, so we are forced to retrieve them all.

I'm not sure if you added this, but it isn't even being used anyway so we can just get rid of it.